sphinx-relay/umbrel-app.yml

```diff
@@ -32,16 +32,24 @@ defaultPassword: ""
 submitter: Stakwork
 submission: https://github.com/getumbrel/umbrel/pull/341
 releaseNotes: >-
-  - Fix: made the bot message seen as false
 
-  - Fix: lower limit amount for /feeds
-
-  - Fix: removed deprecated auth code that was not used in codebase
-
-  - Fix: we updated to a new workflow id
-
-  - Fix: call recoding length
-
-  - And much much more...
-
-  Full changelog can be found here: https://github.com/stakwork/sphinx-relay/releases/tag/v2.2.13
+  - Feat/sentiment bot by @tobi-bams in #633
+  - simplify loadLightning func by @Evanfeenstra in #634
+  - Update/sentiment bot by @tobi-bams in #635
+  - feat: cache stack test by @tobi-bams in #636
+  - Feat/cache integration by @tobi-bams in #639
+  - try fix proxy sub err by @Evanfeenstra in #646
+  - Feat: messages deleter by @tobi-bams in #642
+  - update: add sleep for 10000ms by @tobi-bams in #651
+  - fix: fixing bot message profile picture by @tobi-bams in #652
+  - Feat/javis bot by @tobi-bams in #650
+  - Fix/crash by @tobi-bams in #655
+  - Fix/payment error handling by @tobi-bams in #657
+  - fix grpc ts type generator by @antonilol in #663
+  - CLN native getinfo with generated types by @Evanfeenstra in #664
+  - clearer configs for cln, merge gl and cln cert setup, test cln info by @Evanfeenstra in #665
+  - less strict type guards lnd/proxy/cln by @Evanfeenstra in #670
+  - try catch around massPingHubFromProxies by @Evanfeenstra in #671
+  - whenever check isLND also add or isProxy by @Evanfeenstra in #672
+
+  Full changelog can be found here: https://github.com/stakwork/sphinx-relay/compare/v2.3.0...v2.3.1
```

Comment:
To have these bullet-points show up on separate lines, we need to add an extra newline between each.

Reply:
Actually for updates like this where we are bumping multiple versions AND there are no big new features I think we can just do something like :

```yaml
This release updates Sphinx Relay from version 2.2.13 to 2.3.2.


Full release notes and detailed information is available at https://github.com/stakwork/sphinx-relay/releases
```